### PR TITLE
Adding animations

### PR DESCRIPTION
**Pull request by JewrassicPark**
Updated for latest dev branch

Previous : #178

### CODE REVIEW

**Comment by JewrassicPark** [comment]
I'm not really sure why I had to do this again with the latest dev branch but here it is.

File: src/PhpPresentation/Writer/PowerPoint2007/PptSlides.php [hunk 1]
```diff
@@ -476,7 +476,7 @@ public function writeSlide(Slide $pSlide)
         $objWriter->endElement();
 
         // Loop shapes
-        $shapeId = 0;
+        $shapeId = 1;
```
Comment (Progi1984) [inline src/PhpPresentation/Writer/PowerPoint2007/PptSlides.php]: @JewrassicPark Why do you modify this value ?

Comment (JewrassicPark) [inline src/PhpPresentation/Writer/PowerPoint2007/PptSlides.php]: I answered this in the last fork.

> It seemed like powerpoint was treating shape 0 as the slide itself. IIRC I couldn't get the animations to work when the shape index started at 0.

Comment (Progi1984) [inline src/PhpPresentation/Writer/PowerPoint2007/PptSlides.php]: Oops, I forgot it :)

**Pull request merged**